ember-exclaim/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "ember-exclaim",
+  "version": "1.3.1",
+  "description": "An addon allowing apps to expose declarative, JSON-configurable custom UIs backed by Ember components",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/salsify/ember-exclaim.git",
+    "directory": "ember-exclaim"
+  },
+  "license": "MIT",
+  "author": "Dan Freeman",
+  "files": [
+    "addon-main.cjs",
+    "declarations",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
```

Comment:
Oh neat I was not familiar with concurrently

Reply:
Tbh I find the intermingled output a little rocky sometimes, but it's better than having to spin up separate shells and (more importantly) it's what the blueprint does by default, so I went with it